collection_requirements.rst

```diff
@@ -385,6 +386,5 @@ To be included in the `ansible` package, collections must meet the following cri
 Other things
 ============
 
-* ansible-base's runtime.yml
-* After content is (moved out of community.general or community.network) OR new collection satisfies all the requirements
-    * Add the collection to the ``ansible.in`` file in a corresponding directory of `ansible-build-data repository <https://github.com/ansible-community/ansible-build-data/>`_
+* ansible-core's runtime.yml
```

Comment:
```suggestion
* ansible-core's runtime.yml
```
Can we remove this line?

Reply:
I think so, since changes to it are no longer accepted AFAIK.

Reply:
Removed, thanks